common/setup.cfg

```diff
@@ -12,10 +12,9 @@ long_description_content_type: text/markdown
 classifiers:
     Development Status :: 5 - Production/Stable
     Programming Language :: Python
-    Programming Language :: Python :: 2.7
     Programming Language :: Python :: 3.8
     License :: OSI Approved :: Apache Software License
     Operating System :: OS Independent
 
 [options]
-requires_python: >=2.7, <=3.8, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, !=3.5.*, !=3.6.*, !=3.7.*
+requires_python: >=3.8, <3.9
```

Comment:
I think it some point it might be good to check how the package behaves with pyhton 3.9 and loosen this restriction. Python 3.9 use is going up

Reply:
Thanks @AleksandrLiber , Yes, it will make sense to remove the restriction of 3.9 version in future releases. I have created a JIRA for the same. Please see DLPX-84295